conn.go

```diff
@@ -1055,43 +1055,67 @@ func (c *Conn) Set(path string, data []byte, version int32) (*Stat, error) {
 // same as the input, for example when creating a sequence znode the returned path
 // will be the input path with a sequence number appended.
 func (c *Conn) Create(path string, data []byte, flags int32, acl []ACL) (string, error) {
-	if err := validatePath(path, flags&FlagSequence == FlagSequence); err != nil {
+	createMode, err := parseCreateMode(flags)
+	if err != nil {
 		return "", err
 	}
 
+	if err := validatePath(path, createMode.sequential); err != nil {
+		return "", err
+	}
+
+	if createMode.isTTL {
+		return "", ErrInvalidFlags
```

Comment:
nit: we could wrap these flag errors with more context - eg. `Create` with `TTL` flag is not allowed. This would make it easier to debug the errors for users.